certified-connectors/Yarado/Readme.md

```diff
@@ -0,0 +1,23 @@
+# YARADO
+Yarado automates your admin work so you can move faster.
+
+
+## Publisher: Yarado
+
+
+## Prerequisites
+### A Microsoft Power Apps or Power Automate plan
+### A Yarado account
+To use this connector, you will need an Yarado account. [Contact us](https://www.yarado.com/) for more information.
+
+
```

Comment:
@MarcinKowalczukYarado please make sure that all of the required headings from [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1) are present